flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/strategy/LazyFromSourcesSchedulingStrategyTest.java

```diff
@@ -343,31 +365,9 @@ public void testBlockingANYExecutionStateChange() {
 			schedulingStrategy.onExecutionStateChange(producer.getId(), ExecutionState.FINISHED);
 		}
 
-		assertThat(testingSchedulerOperation, hasScheduledVertices(consumers));
-	}
-
-	private static Matcher<TestingSchedulerOperations> hasScheduledVertices(final List<TestingSchedulingExecutionVertex> consumers) {
-
-		final Matcher<Iterable<? extends ExecutionVertexID>> vertexIdMatcher = containsInAnyOrder(consumers.stream()
```

Comment:
Replacing with `Matchers.contains` didn't work?

Reply:
I think equals is simpler and then we do not need to keep this Matcher.